### PR TITLE
Updating cli docs for octopus.server.exe

### PR DESCRIPTION
**Pull request by Octobob**
An automated update of the command line docs for octopus.server.exe version 2020.1.12.

Created by TeamCity project 'Automated Documentation Updates::Update CLI Docs', build 207

### CODE REVIEW

File: docs/octopus-rest-api/octopus.server.exe-command-line/index.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ hideInThisSection: true
 - **[import-certificate](/docs/octopus-rest-api/octopus.server.exe-command-line/import-certificate.md)**:  Replace the certificate that Octopus Server uses to authenticate itself with its Tentacles.
 - **[license](/docs/octopus-rest-api/octopus.server.exe-command-line/license.md)**:  Import a license key.
 - **[list-instances](/docs/octopus-rest-api/octopus.server.exe-command-line/list-instances.md)**:  Lists all installed Octopus instances.
-- **[lost-master-key](/docs/octopus-rest-api/octopus.server.exe-command-line/lost-master-key.md)**:  Get your Octopus Server working again after losing your Master Key.
+- **[lost-master-key](/docs/octopus-rest-api/octopus.server.exe-command-line/lost-master-key.md)**:  Get your Octopus Server working again after losing your master key.
```
Comment (matt-richardson) [inline docs/octopus-rest-api/octopus.server.exe-command-line/index.md:25]: @wordlee - looks like this got fixed and then reverted?
https://github.com/OctopusDeploy/OctopusDeploy/commit/a1c9a09b7d2995e3e8572ca45edf732a207a7e81

File: docs/octopus-rest-api/octopus.server.exe-command-line/configure.md [hunk 2]
```diff
@@ -69,9 +69,14 @@ Where [<options>] is any of:
                              Comma-separated whitelist of domains that are
                                allowed to retrieve data (empty turns CORS off,
                                * allows all).
+      --xFrameOptions=VALUE  A directive to provide in the X-Frame-Options
```
Comment (matt-richardson) [inline docs/octopus-rest-api/octopus.server.exe-command-line/configure.md:72]: Not sure why this has disappeared. Having trouble keeping track of the [reverts of the reverts](https://github.com/OctopusDeploy/OctopusDeploy/commit/df27c9975bd43ab4007db2d7fdbec2411f82acc7).
Maybe @alialmaktoum can provide some insight into why it's not `2020.1.12`?

File: docs/octopus-rest-api/octopus.server.exe-command-line/configure.md [hunk 3]
```diff
@@ -196,7 +203,7 @@ Where [<options>] is any of:
                              Set whether GitHub issue tracker integration is
                                enabled.
       --GitHubBaseUrl=VALUE  Set the base url for the Git repositories.
-      --jiraIsEnabled=VALUE  Set whether Jira Integration is
+      --jiraIsEnabled=VALUE  Set whether Jira issue tracker integration is
```
Comment (matt-richardson) [inline docs/octopus-rest-api/octopus.server.exe-command-line/configure.md:206]: Ahh, looks like @Silcoish made this [change manually](https://github.com/OctopusDeploy/docs/commit/0309a554726d5f75bd8732fb7fcae292c3fd90f6)...
What version is this available to public, @Silcoish?

File: docs/octopus-rest-api/octopus.server.exe-command-line/configure.md [hunk 4]
```diff
@@ -123,7 +128,9 @@ Where [<options>] is any of:
                              When Domain authentication is used, specifies
                                the scheme (Basic, Digest,
                                IntegratedWindowsAuthentication, Negotiate,
-                               Ntlm).
+                               Ntlm). You will need to restart all Octopus
```
Comment (matt-richardson) [inline docs/octopus-rest-api/octopus.server.exe-command-line/configure.md:131]: This feels like a change we want 👍

File: docs/octopus-rest-api/octopus.server.exe-command-line/index.md (hunk 1 above)
Comment (wordlee) [reply docs/octopus-rest-api/octopus.server.exe-command-line/index.md:25]: Yeah, that was late on a Friday and I was worried about failing test over the weekend. I'm redoing it now and will tag you on the PR.

File: docs/octopus-rest-api/octopus.server.exe-command-line/configure.md (hunk 3 above)
Comment (Silcoish) [reply docs/octopus-rest-api/octopus.server.exe-command-line/configure.md:206]: 2020.2.0

**Pull request closed without merging**